Logo.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { StyleSheet, Text, View, Button } from 'react-native';
+import { Font } from 'expo';
+
+
+export default class Logo extends React.Component {
+  render() {
+    return (
+      <View style = {{
+        height: '30%',
+        flexDirection: 'column',
+        marginTop: '13%',
+      }}>
+        <View style={[styles.logo, styles.pong]} />
+        <View style={[styles.logo, styles.mates, {marginTop: 30}]} />
+        <View style={[styles.logo, styles.pong, styles.cover, {marginTop: 7}]} />
+        <View style={[styles.logo, styles.mates, styles.cover, {marginTop: 37}]} />
+        {
+          this.props.font ? (
+          <View style={{
+            alignSelf: 'center',
+            justifyContent: 'center',
+            flexDirection: 'row',
+            position: 'absolute',
+            marginTop: 10,
+          }}>
+            <Text style={styles.pongMatesText}>
+            Pong
+            </Text>
+            <Text style={[styles.pongMatesText, {marginTop: 30}]}>
+            Mates
+            </Text>
+          </View>
+          ) : null
```

Comment:
What happens if `this.props.font` comes out to be `false`? Nothing show up? Make sure that you have back up text